app/code/Magento/VaultGraphQl/Test/Unit/Model/Resolver/DeletePaymentTokenTest.php

```diff
@@ -0,0 +1,232 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\VaultGraphQl\Test\Unit\Model\Resolver;
+
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlNoSuchEntityException;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\GraphQl\Model\Query\ContextInterface;
+use Magento\GraphQl\Model\Query\ContextExtensionInterface;
+use Magento\Vault\Api\PaymentTokenManagementInterface;
+use Magento\Vault\Api\PaymentTokenRepositoryInterface;
+use Magento\Vault\Api\Data\PaymentTokenInterface;
+use Magento\VaultGraphQl\Model\Resolver\DeletePaymentToken;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test class for \Magento\VaultGraphQl\Model\Resolver\DeletePaymentToken
+ */
+class DeletePaymentTokenTest extends TestCase
+{
+    /**
+     * Object Manager Instance
+     *
+     * @var ObjectManager
+     */
+    private $objectManager;
+
+    /**
+     * Testable Object
+     *
+     * @var DeletePaymentToken
+     */
+    private $resolver;
+
+    /**
+     * @var ResolverInterface|MockObject
+     */
+    private $resolverMock;
+
+    /**
+     * @var ContextInterface|MockObject
+     */
+    private $contextMock;
+
+    /**
+     * @var ContextExtensionInterface|MockObject
+     */
+    private $contextExtensionMock;
+
+    /**
+     * @var Field|MockObject
+     */
+    private $fieldMock;
+
+    /**
+     * @var PaymentTokenManagementInterface|MockObject
+     */
+    private $paymentTokenManagementMock;
+
+    /**
+     * @var PaymentTokenRepositoryInterface|MockObject
+     */
+    private $paymentTokenRepositoryMock;
+
+    /**
+     * @var PaymentTokenInterface|MockObject
+     */
+    private $paymentTokenMock;
+
```

Comment:
Please, make sure you don't have duplicated empty lines.

Reply:
Yep... **fixed**